asv/graph.py

```diff
@@ -118,13 +118,15 @@ def resample_data(self, val):
 
         new_val = []
         j = 0
-        for i in xrange(min_time + step_size, max_time + step_size, step_size):
+        i = min_time + step_size
```

Comment:
Why this change?  Less likely for a change to inadvertently fail the loop end condition when using `xrange`...